packages/mwp-app-render/src/util/seo/__snapshots__/ldJson.test.js.snap

```diff
@@ -31,7 +31,7 @@ Object {
     "price": 10,
     "priceCurrency": "USD",
   },
-  "startDate": "1970-01-18T04:01-05:00",
+  "startDate": "1970-01-18T05:01-04:00",
```

Comment:
hmm i wonder why this changed in the snap?

Reply:
it's because of my timezone. I'll change it back